.woodpecker.yml

```diff
@@ -212,7 +213,7 @@ steps:
     when: *slow_check_paths
 
   run_federation_tests:
-    image: node:24-trixie-slim
+    image: node:current-slim
```

Comment:
This might fail federation tests. I remember having issues earlier when not using the trixie image.